src/tabs/interfaces.ts

```diff
@@ -90,6 +91,24 @@ export namespace TabsProps {
      * Whether this tab is disabled.
      */
     disabled?: boolean;
+    /**
+     * (Optional) Determines whether the tab includes a close icon button. By default, the close button is not included.
```

Comment:
(here and throughout) maybe just "close button" rather than "close icon button". Or do we want to consistently refer to this as "dismiss button" to be consistent with the API? Depends a bit what terminology we use in the usage guidelines too